replay_build_scripts/updateRepo.js

```diff
@@ -0,0 +1,59 @@
+const { syncRepo } = require("./syncRepo");
+
+export function updateRepo() {
```

Comment:
Again, probably good to keep this next to syncRepo, assert, and spawnChecked functions.  Also, I'd love to rename this to "updateAllRepos".